apis/core/v1beta1/openstackcontrolplane_types.go

```diff
@@ -484,10 +489,32 @@ type RabbitmqSection struct {
 
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
-	// Templates - Overrides to use when creating the Rabbitmq clusters
+	// Templates - Overrides to use when creating the Rabbitmq clusters for RPC and (optionally) Notifications.
 	Templates *map[string]rabbitmqv1.RabbitMqSpecCore `json:"templates"`
 }
 
+// NotificationsBusSection defines the desired state of AMQP messaging Bus Services for producers and consumers of notifications
+type NotificationsBusSection struct {
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=true
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:booleanSwitch"}
+	// Enabled - Whether Notifications Bus Service should be configured for all services that need it.
+	// When enabled, leaves a possibility for each service to override the configuration, or disable notifications only for itself.
+	// When disabled, leaves the notification drivers of all services unconfigured and emitting no notifications at all.
```

Comment:
@gibizer , wrt:

> but allow to override on NovaSpecCore level with a value or with ""
> if NovaSpecCore level is set to "", then do not emit notifications regardless of the global rabbit > value.
> if NovaSpecCore level is et to "foo", then emit notification to bus foo regardless of the global rabbit value.

I decided to change the former one in this proposal, let's see how we want it, and adjust